persistence-modules/hibernate-mapping/src/main/java/com/baeldung/hibernate/pojo/EntityDescription.java

```diff
@@ -14,10 +17,12 @@ public class EntityDescription implements Serializable {
 
     private String description;
 
-    @Any(
-            metaDef = "EntityDescriptionMetaDef",
-            metaColumn = @Column(name = "entity_type")
-    )
+    @Any
+    @AnyDiscriminator(DiscriminatorType.STRING)
+    @AnyDiscriminatorValue(discriminator = "S", entity = Employee.class)
```

Comment:
can you let me know with which article this is linked?

Reply:
It is related to this article: https://www.baeldung.com/hibernate-dynamic-mapping

Reply:
Sure, I will update all the articles presented in all modules that I made changes.